### ISSUE TITLE
Code review button doesn't work on commits.

### ISSUE THREAD

**Issue by wyattoday** [go-gitea/gitea#4625]
- Gitea version (or commit ref): https://github.com/go-gitea/gitea/commit/7cb1c1cf20f1a07d98b78c2ba56c3b50777701a5
- Git version: 2.18.0
- Operating system: Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [X] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [X] Yes: https://try.gitea.io/wyattoday/Test1234/commit/31fb0641bc7f5d7f5421ec13790f8252c6439c1c
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

New code-review feature doesn't work. This merged pull request: https://github.com/go-gitea/gitea/pull/3748

Click the "Review" button and nothing comes up. Neither does clicking the the lines to make code comments.

Presumably this is an issue with the JavaScript.

**Comment by jonasfranz** [comment]
You mean the '+' button at the diff view, doesn't you? The button shouldn't be shown at all. We currently donnot provide comments on commits at all. (Just comments on pulls)

**Comment by wyattoday** [comment]
If you're logged in (and presumably have permission for the repository) then on a commit there's a green "Review" button. That button doesn't do anything.

My impression was that this commit added commenting on commits: https://github.com/go-gitea/gitea/pull/3748

(Because it shows comment on commits).

So, either the "Review" button shouldn't be visible because that functionality hasn't been added yet, or the functionality has been added but the "Review" button is broken.

I was able to reproduce this with the latest Gitea commits and on try.gitea.io (see the above link, or view a commit you have permission to and see the green non-functional "Review" button)
